### ISSUE TITLE
Uncaught TypeError: Cannot read property 'height' of undefined

### ISSUE THREAD

**Issue by davidfowl** [OmniSharp/omnisharp-atom#149]
[Enter steps to reproduce below:]

1. Try to change the height of the panel.

**Atom Version**: 0.194.0
**System**: Mac OS X 10.10.2
**Thrown From**: [omnisharp-atom](https://github.com/OmniSharp/omnisharp-atom) package, v0.4.1

### Stack Trace

Uncaught TypeError: Cannot read property 'height' of undefined

```
At /Users/dfowler/.atom/packages/omnisharp-atom/lib/omnisharp-atom/views/dock-view.ts:32

TypeError: Cannot read property 'height' of undefined
  at DockView.resizeStarted (/Users/dfowler/.atom/packages/omnisharp-atom/lib/omnisharp-atom/views/dock-view.ts:32:64)
  at HTMLDivElement.<anonymous> (/Users/dfowler/.atom/packages/omnisharp-atom/lib/omnisharp-atom/views/dock-view.ts:145:92)
  at space-pen-div.jQuery.event.dispatch (/Users/dfowler/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4430:9)
  at space-pen-div.elemData.handle (/Users/dfowler/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4116:28)

```

### Commands

```
  2x -8:08.5.0 core:move-right (atom-text-editor.editor)
     -8:05.9.0 core:save (atom-text-editor.editor)
     -8:05.6.0 core:move-up (atom-text-editor.editor)
     -8:04.6.0 core:move-down (atom-text-editor.editor)
  2x -8:04.3.0 core:save (atom-text-editor.editor)
     -0:44.9.0 core:move-up (atom-text-editor.editor)
     -0:44.5.0 editor:move-to-end-of-screen-line (atom-text-editor.editor)
  2x -0:44.2.0 core:select-up (atom-text-editor.editor)
     -0:43.7.0 editor:select-to-first-character-of-line (atom-text-editor.editor)
  6x -0:43.4.0 core:backspace (atom-text-editor.editor)
     -0:39.7.0 core:move-right (atom-text-editor.editor)
  3x -0:37.3.0 core:move-down (atom-text-editor.editor)
     -0:36.6.0 core:move-left (atom-text-editor.editor)
  6x -0:36.4.0 core:backspace (atom-text-editor.editor)
     -0:34 autocomplete-plus:confirm (atom-text-editor.editor)
     -0:33.8.0 core:save (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v2.12.0
language-csharp, v0.5.0
linter, v0.12.1
omnisharp-atom, v0.4.1

# Dev
No dev packages
```

**Comment by nosami** [comment]
Looks like a dupe of https://github.com/OmniSharp/omnisharp-atom/issues/139. Is this still causing you problems? I thought this was fixed in 0.4.1 /cc @david-driscoll

**Comment by davidfowl** [comment]
Fixed in the latest
